### ISSUE TITLE
Please return a flag for a custom license in the add-on detail API response

### ISSUE THREAD

**Issue by bobsilverberg** [mozilla/addons-server#9914]
Following up on https://github.com/mozilla/addons-frontend/issues/3697#issuecomment-436254019, we currently get the following for the license in `current_version` from the [add-on detail API](https://addons-server.readthedocs.io/en/latest/topics/api/addons.html#detail):
- id
- name
- url

If the license is a custom license (i.e., the text of the license is stored with the version information), then the only indication we have of that is that the `url` points back to the app (e.g., `https://addons-dev.allizom.org/en-US/firefox/addon/adblock-edge/license/1.6b`). 

We would like to also get an explicit indication that the version contains a custom license in this case. This could mean adding a property to `current_version` for `has_custom_license` or something along those lines.

**Comment by bobsilverberg** [comment]
Please note that https://github.com/mozilla/addons-frontend/issues/3697 is meant to land this quarter, so it would be appreciated if this could be prioritized accordingly.

**Comment by AlexandraMoga** [comment]
The add-on details endpoint `licence` information is now including a `is_custom` property:

https://addons-dev.allizom.org/api/v4/addons/addon/greatly-for-firefox/

![image](https://user-images.githubusercontent.com/31961530/48341076-d4778e80-e674-11e8-8431-090a6f223848.png)

@eviljeff I've checked several add-ons for which I've added a specific Custom licence name and text, either in the submission flow or in Dev Hub Edit authors and licence information, but for all these examples the `"licence": {..."name"` is returned `null` by the api.
I've tried the same thing for stage but there the api is returning my custom licence name. 
probably, this needs a follow up issue. 

See https://addons.allizom.org/api/v4/addons/addon/devhub-release-08-11/
